site/team/presse/index.html

```diff
@@ -0,0 +1,15 @@
+<meta http-equiv="refresh" content="5; url=https://www.fetzer-bau.de/" />
```

Comment:
...

Reply:
This is my purpose, I won't betray my god.